File: travelbookwithJava/app/src/main/java/com/abdbzkn/travelbookwithjava/roomdb/PlaceDatabase.java
package com.abdbzkn.travelbookwithjava.roomdb;

import androidx.room.Database;
import androidx.room.RoomDatabase;
import com.abdbzkn.travelbookwithjava.model.Place;

@Database(entities = {Place.class},version = 1)
public abstract class PlaceDatabase extends RoomDatabase {
    public abstract PlaceDao placedao();
}

File: travelbookwithKotlin/app/src/main/java/com/abdbzkn/travelbookwithkotlin/roomdb/PlaceDatabase.kt
package com.abdbzkn.travelbookwithkotlin.roomdb

import androidx.room.Database
import androidx.room.RoomDatabase
import com.abdbzkn.travelbookwithkotlin.model.Place

@Database(entities = [Place::class], version = 1)
abstract class PlaceDatabase : RoomDatabase() {
    abstract fun placeDao(): PlaceDao
}